using.rst

```diff
@@ -424,6 +424,8 @@ Tag expressions
 
 Tags can be selected using expressions. Examples:
 
+(NOTE: "!" is a reserved bash character, you may need to escape it using "\!")
```

Comment:
Please use the RST [`note` admonition](http://docutils.sourceforge.net/docs/ref/rst/directives.html#admonitions) directive. Example: https://github.com/getgauge/docs.getgauge.io/blame/master/quickstart.rst#L34